hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -454,6 +454,11 @@ private OzoneManager(OzoneConfiguration conf, StartupOption startupOption)
     this.omNodeDetails = omhaNodeDetails.getLocalNodeDetails();
 
     omStorage = new OMStorage(conf);
+    // validate if the stored one is the same we get in the OMNodeDetails
+    // if not, fails the start. If the VERSION file does not contain the node
+    // id yet, it stores it to the VERSION file and skips validation for the
+    // that case.
```

Comment:
```suggestion
    // validate if the stored node ID is the same we get in the OMNodeDetails
    // if not, fails to start. If the VERSION file is from an older Ozone version and does not contain the node
    // ID yet, it stores it to the VERSION file and skips validation.
```

Minor suggestion here to clarify why the version file may not have the node ID and we need to write it. If you just read the code after this change, it looks like the node ID should show up on init and the constructor can assume it is already there. It is not obvious that the node ID used to not be written in init.

Reply:
Based on the previous comment where you seemed to agree that the method apidoc is enough, I removed this part, and extended the apidoc a bit.